ch04.adoc

```diff
@@ -55,11 +51,11 @@ float lat(lat) ;
 
 ====
 
-Application writers should note that the Udunits package does not recognize the directionality implied by the "north" part of the unit specification.
+Application writers should note that the UDUNITS package does not recognize the directionality implied by the "north" part of the unit specification.
 It only recognizes its size, i.e., 1 degree is defined to be pi/180 radians.
 Hence, determination that a coordinate is a latitude type should be done via a string match between the given unit and one of the acceptable forms of **`degrees_north`**.
 
-Optionally, the latitude type may be indicated additionally by providing the **`standard_name`** attribute with the value **`latitude`**, and/or the **`axis`** attribute with the value **`Y`**.
+It is strongly recommended that the latitude type is also indicated by providing the **`standard_name`** attribute with the value **`latitude`**, and/or the **`axis`** attribute with the value **`Y`**.
```

Comment:
I think this change (and the similar change to the longitude section), is substantive and also requires an update to the conformance document. So I shall revert this line back to the original (which is nice and easy now that one sentence per line!)

Reply:
I agree that it is a substantive change , so yes remove it. The question then is if the change is a good one that should be picked up in a separate discussion issue?